modules/application-health-configuring.adoc

```diff
@@ -5,45 +5,86 @@
 [id="application-health-configuring_{context}"]
 = Configuring health checks using the CLI
 
-To configure health checks, create a pod for each type of check you want.
+To configure Readiness, Liveness, and Startup probes, create a pod for each type of probe you want. You can add multiple probes to a single pod or create separate pods for each probe.
```

Comment:
I'd revise this to reflect that the probes for a pod live in the spec for that pod.